src/Exceptions/InvalidJSONFormatException.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+namespace RonasIT\Support\Exceptions;
+
+use Exception;
+
+class InvalidJSONFormatException extends Exception
+{
+}
```

Comment:
let's extend the message of custom exception

`Attempt to read non json response from the {$requestType} request on {$url}. Received response: "{$actualResponse}".`

Reply:
Done, but I keep only `$actualResponse` because we always know what were the request type and url